packages/server/graphql/types/UpdatePromptResponsePayload.ts

```diff
@@ -0,0 +1,31 @@
+import {GraphQLObjectType} from 'graphql'
+import {GQLContext} from '../graphql'
+import {makeResolve, resolveNewMeeting} from '../resolvers'
+import makeMutationPayload from './makeMutationPayload'
+import NewMeeting from './NewMeeting'
+import StandardMutationError from './StandardMutationError'
+import TeamPromptResponse from './TeamPromptResponse'
+
+export const UpdatePromptResponseSuccess = new GraphQLObjectType<any, GQLContext>({
+  name: 'UpdatePromptResponseSuccess',
+  fields: () => ({
+    error: {
+      type: StandardMutationError
+    },
+    meeting: {
```

Comment:
+1 I'd also add a `meetingId` to `UpdatePromptResponseSuccess ` as it most probably will be needed in updaters on frontend side, we might get away without fetching the whole meeting